packages/manager/apps/support/index.js

```diff
@@ -0,0 +1,29 @@
+import support from '@ovh-ux/manager-support';
+import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
+import uiRouter from '@uirouter/angularjs';
+import angular from 'angular';
+
+/* eslint-disable import/no-webpack-loader-syntax, import/extensions */
+import 'script-loader!jquery';
+import 'script-loader!lodash';
```

Comment:
Is it required as you import lodash functions one by one ?

Reply:
Let's find out 💪